student.rb

```diff
@@ -1,15 +1,12 @@
-class Student
+require_relative 'people'
+
+class Student < People
```

Comment:
A Student is a People? Make sure the pluralization matches (so use `Person` instead of `People` for the superclass.